src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -41,16 +41,21 @@ public CommandResult execute() {
     }
 
     /**
-     * Retrieves all persons in the address book whose names contain some of the specified keywords.
+     * Retrieves all persons in the address book whose names contain some of the specified keywords
+     * ,regardless of case of letters.
      *
      * @param keywords for searching
      * @return list of persons found
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            Set<String> lowerCaseWordsInName = new HashSet<>();
+            for(String word: wordsInName){
+                lowerCaseWordsInName.add(word.toLowerCase());
+            }
+            if (!Collections.disjoint(lowerCaseWordsInName, keywords)) {
```

Comment:
There's an implicit assumption that keywords will always be in lower-case. What if `find JOHN` was called?